### PR TITLE
Add tests for container type reflection attributes

### PR DESCRIPTION
**Pull request by toji**
Adds tests for Texture, Buffer, and QuerySet to ensure that their
reflection attributes are set properly upon construction.

These properties aren't currently implemented in Chrome, but either I or another team member are likely to get to that shortly. It's fine by me to hold off on landing this until then.
<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [ ] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/api/operation/reflection.spec.ts
Comment (kainino0x) [inline src/webgpu/api/operation/reflection.spec.ts:13]: optionally:
```
  .paramsSubcasesOnly(u =>
    u.combine('descriptor', [
```

**Review by kainino0x — APPROVED** [review]

File: src/webgpu/api/operation/reflection.spec.ts
Comment (toji) [reply src/webgpu/api/operation/reflection.spec.ts]: Done! Going to wait till https://chromium-review.googlesource.com/c/chromium/src/+/3687793 lands to verify the test before landing, though.

**Comment by toji** [comment]
Testing shows this works in Chrome now, so merging.

**Pull request merged**
